docs/src/main/asciidoc/observability-devservices-lgtm.adoc

```diff
@@ -0,0 +1,281 @@
+----
+
+And then explicitly list LGTM Dev Resource in the test as a `@QuarkusTestResource` resource:
+[source, java]
+----
+@QuarkusTest
+@QuarkusTestResource(value = LgtmResource.class, restrictToAnnotatedClass = true)
+@TestProfile(QuarkusTestResourceTestProfile.class)
+public class LgtmLifecycleTest extends LgtmTestBase {
+}
+----
+
+== Testing full Grafana OTel LGTM stack - example
+
+Use existing Quarkus MicroMeter OTLP registry
+
+[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
+.pom.xml
+----
+<dependency>
+    <groupId>io.quarkiverse.micrometer.registry</groupId>
+    <artifactId>quarkus-micrometer-registry-otlp</artifactId>
+</dependency>
+----
+
+[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
+.build.gradle
+----
+implementation("io.quarkiverse.micrometer.registry:quarkus-micrometer-registry-otlp")
+----
+
+On the test  `application.properties` file, you need to define:
+[source,properties]
+----
+# Micrometer OTLP registry
+quarkus.micrometer.export.otlp.url=http://${quarkus.otel-collector.url}/v1/metrics
+# OpenTelemetry
+quarkus.otel.exporter.otlp.traces.protocol=http/protobuf
+quarkus.otel.exporter.otlp.traces.endpoint=http://${quarkus.otel-collector.url}
+----
+
+Simply inject the Meter registry into your code -- it will periodically push metrics to Grafana LGTM's OTLP HTTP endpoint.
+
+[source, java]
+----
+@Path("/api")
+public class SimpleEndpoint {
+    private static final Logger log = Logger.getLogger(SimpleEndpoint.class);
+
+    @Inject
+    MeterRegistry registry;
+
+    @PostConstruct
+    public void start() {
+        Gauge.builder("xvalue", arr, a -> arr[0])
+                .baseUnit("X")
+                .description("Some random x")
+                .tag("my_key", "x")
+                .register(registry);
+    }
+
+    // ...
+}
+----
+
+Where you can then check Grafana's datasource API for existing metrics data.
+
+[source, java]
+----
+public class LgtmTestBase {
+
+    @ConfigProperty(name = "quarkus.grafana.url")
+    String url; // NOTE -- injected Grafana endpoint url!
+
+    @Test
+    public void testTracing() {
+        String response = RestAssured.get("/api/poke?f=100").body().asString();
+        System.out.println(response);
+        GrafanaClient client = new GrafanaClient("http://" + url, "admin", "admin");
+        Awaitility.await().atMost(61, TimeUnit.SECONDS).until(
+                client::user,
+                u -> "admin".equals(u.login));
+        Awaitility.await().atMost(61, TimeUnit.SECONDS).until(
+                () -> client.query("xvalue_X"),
+                result -> !result.data.result.isEmpty());
+    }
+
+}
+
+// simple Grafana HTTP client
+
+public class GrafanaClient {
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    private final String url;
+    private final String username;
+    private final String password;
+
+    public GrafanaClient(String url, String username, String password) {
```

Comment:
We are missing classes in the client and the response doesn't work with standards micrometer metrics

Reply:
> We are missing classes in the client

Which classes?

> response doesn't work with standards micrometer metrics

Yeah, a few more `ignore on unknown property = false` ...

Reply:
In the documentation, we miss `QueryResult` and `User`, I believe.